src/MicrosoftTeams.ts

```diff
@@ -182,6 +374,8 @@ namespace microsoftTeams {
   let backButtonPressHandler: () => boolean;
   handlers["backButtonPress"] = handleBackButtonPress;
 
+  window.onNativeMessage = handleParentMessage;
```

Comment:
Please move this into the initialize() API and only set it when you detect that you’re running in frameless mode (i.e. when you don’t have a parent or opener).

Reply:
done